fast/stages/3-network-security/variables.tf

```diff
@@ -0,0 +1,38 @@
+/**
+ * Copyright 2024 Google LLC
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+variable "factories_config" {
+  description = "Configuration for network resource factories."
+  type = object({
+    data_dir = optional(string, "data")
```

Comment:
nit: this should be named after the resource type and point to the folder inside data (so one level deeper)

Reply:
and it looks like dev and prod folders are both needed, so I would do something like

```
type = object({
  policy_rules = optional(object({
    dev = string
```